README.md

```diff
@@ -43,93 +58,124 @@ interface reactDesc {
 stamp.compose(...desc?: stamp|reactDesc|specDesc[]): stamp
 ```
 
-The most powerful feature of [stamps](https://github.com/stampit-org/stamp-specification) is their composability. What this means is that `n` number of stamps can be combined into a new stamp which inherits each passed stamp's behavior. This is perfect for React, since `class` is being pushed as the new norm and does not provide an idiomatic way to use mixins. (classical inheritance :disappointed:). Stamps embrace prototypal inheritance and as a result provide great flexibility when extending React components.
+The most powerful feature of [stamps](https://github.com/stampit-org/stamp-specification) is their composability. Any number of stamps can be combined into a new stamp which inherits each passed stamp's behavior. This is perfect for React since `class` is being pushed as the new norm and does not provide an idiomatic way to use mixins. (classical inheritance :disappointed:). Stamps embrace prototypal inheritance and as a result provide great flexibility when composing React components.
 
-__reactStamp.js__
+Let's go step by step through an example. It uses a pattern I call **Behavior Driven Composition**.
+
+__container.js__
 ```js
-import React from 'react';
-import reactStamp from 'react-stamp';
+container(React: object, components: object): componentDescriptor
+```
+```js
+export default (React, { Button, Text }) => ({
+  init() {
+    this.behaviors = [];
+    this.state = { showBehaviors: false };
+  },
+
+  render() {
+    const { showBehaviors } = this.state;
+    const behaviors = showBehaviors && this.behaviors.toString();
 
-export default reactStamp(React);
+    return (
+      <div>
+        <Button
+          value='Show Behaviors'
+          onClick={() => this._onClick()}
```

Comment:
@koresar 
> If a behaviour/component depends on something it should "require" (or load somehow) the missing bit. That's also IMO.

How would you recommend handling this click handler? It won't be defined until the behavior mixins are inherited.